src/cdk/listbox/listbox.ts

```diff
@@ -418,6 +419,13 @@ export class CdkListbox<T = unknown> implements AfterContentInit, OnDestroy, Con
   /** Whether the listbox currently has focus. */
   private _hasFocus = false;
 
+  /** A reference to the option that was active before the listbox lost focus. */
+  private _previousActiveOption: CdkOption<T> | null = null;
+
+  constructor(private _ngZone: NgZone) {
```

Comment:
This should use the `inject` function rather than constructor-based injection. We're trying to use it for all new code.